src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import { View } from 'react-native';
+
+import { StoryContainer } from '../../common/HelperComponents/StoryContainer';
+import { Badge } from './Badge';
+
+const BadgesContainer = styled.View`
+  flex-direction: row;
+  justify-content: flex-start;
+`;
+
+const badgeDefinition: string = 'Notify the user by calling their attention to an update or something important';
+
+export default {
+  component: Badge,
+  parameters: {
+    componentSubtitle: badgeDefinition,
+  },
```

Comment:
falta adicionar o status do componente com o que foi contemplado e o que nao foi de acordo com a doc de ux

Reply:
@Michelly-Oliveira recentemente a gente adotou um padrão para as documentações mostrarem no topo o status de completude, como esse [exemplo aqui](https://github.com/natura-cosmeticos/natds-rn/blob/master/src/components/Button/Button.stories.tsx)

os itens da lista e a ordem seguem o [ZeroHeight](https://zeroheight.com/28db352be/p/920945-badge/b/27ccda) pois é lá que as pessoas olham esse status.

Reply:
o status da badge seria mais ou menos assim:
 ```
const description = () => `
---
**NOTE**: This component is available in the following variants:
  - ❌  Dot
  - ❌  Pulse
  - ❌  Standard
With the following attribute status:
- **Color:**
    - ❌  \`alert\`
    - ❌  \`primary\`
    - ❌  \`secondary\`
    - ❌  \`success\`
- **Limit**
  - ❌  \`9\`
  - ❌  \`99\`
  - ❌  \`unlimited\`
---
`;
 ```

aí você coloca como descrição no `export default` como tá no exemplo que eu mandei nas linhas 42~45.

foi mal o comentário longo! qualquer coisa to a disposição ✌️